source/module_elecstate/test/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ AddTest(
 AddTest(
   TARGET elecstate_print
   LIBS ${math_libs} base device
-  SOURCES elecstate_print_test.cpp ../elecstate_print.cpp ../occupy.cpp
+  SOURCES elecstate_print_test.cpp ../elecstate_print.cpp ../occupy.cpp ../../module_io/json_output/abacusjson.cpp  ../../module_io/json_output/output_info.cpp
```

Comment:
please remove the ../../module_io/json_output/xxx.cpp

Reply:
remove these now.